1-quickstart/README.md

```diff
@@ -52,7 +55,96 @@ docker build -t mass-ingest:quickstart --build-arg MODERNE_CLI_VERSION=3.50.0 ..
 
 ### 3. Run the container
 
-Credentials are configured at runtime (not baked into the image):
+Credentials are configured at runtime (not baked into the image). You can use S3, Artifactory, or any Maven-compatible repository.
+
+#### Option A: Using S3 Storage
+
+For S3 or S3-compatible storage, the Moderne CLI supports all standard AWS credential providers.
+
+**S3 Configuration Variables:**
+- `PUBLISH_URL` - S3 bucket URL (must start with `s3://`)
+- `S3_PROFILE` - AWS profile name (optional, uses AWS credential chain by default)
+- `S3_REGION` - AWS region (optional, for cross-region bucket access)
+- `S3_ENDPOINT` - S3 endpoint URL (optional, for S3-compatible services like MinIO)
+
+**AWS Authentication:**
+The container supports all standard AWS authentication methods:
+- IAM roles (automatic on EC2/ECS/Fargate)
+- Environment variables (`AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`)
+- AWS credentials file with profiles
+- AWS SSO
+- For detailed options, see [AWS CLI Configuration documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html)
+
+**Option 1: Using AWS Environment Variables (Simplest)**
```

Comment:
```suggestion
**Example 1: Using AWS Environment Variables (Simplest)**
```

Just trying to be clear that these are examples rather than explicitly necessary options.

Reply:
For me, I read "option" as a specific choice, but "example" as an illustration.

Reply:
Indeed, each customer will make a choice of how they're going to provide the credentials to mass ingest. Some customers may have multiple options, some may only be able to use one or the other.